Sources/StreamChat/Models/UnreadCount.swift

```diff
@@ -4,6 +4,10 @@
 
 import Foundation
 
+// TODO: v5: Because we mix the UnreadCount Domain model with Payload we have to set `threads` optional.
```

Comment:
Maybe add this in the notion v5 document instead? We will probably not see/open this anymore and likely forget about it 😅

Reply:
The goal would be to search in the code for TODO and v5, then we can remind us about things that need to be done, but I'll add it to notion as well 👍